tests/foreman/api/test_errata.py

```diff
@@ -17,16 +17,159 @@
+                self.assertEqual(result.return_code, 0)
+                result = client.run('rpm -q {}'.format(package_name))
+                self.assertEqual(result.return_code, 0)
+            return
+        entities.Host().install_content(data={
+            'organization_id': self.org.id,
+            'included': {'ids': host_ids},
+            'content_type': 'package',
+            'content': [package_name],
+        })
+        self._validate_package_installed(clients, package_name)
+
+    def _validate_package_installed(self, hosts, package_name,
+                                    expected_installed=True, timeout=120):
+        """Check whether package was installed on the list of hosts."""
+        for host in hosts:
+            for _ in range(timeout / 15):
+                result = host.run('rpm -q {0}'.format(package_name))
+                if (result.return_code == 0 and expected_installed or
+                        result.return_code != 0 and not expected_installed):
+                    break
+                sleep(15)
+            else:
+                self.fail(
+                    u'Package {0} was not {1} host {2}'.format(
+                        package_name,
+                        'installed on' if expected_installed else
+                        'removed from',
+                        host.hostname,
+                    )
+                )
+
+    def _validate_errata_counts(self, host, errata_type, expected_value,
+                                timeout=120):
+        """Check whether host contains expected errata counts."""
+        for _ in range(timeout / 5):
+            host = host.read()
+            if (host.content_facet_attributes[
+                    'errata_counts'][errata_type] == expected_value):
+                break
+            sleep(5)
+        else:
+            self.fail(
+                u'Host {0} contains {1} {2} errata, but expected to contain '
+                '{3} of them'.format(
+                    host.name,
+                    host.content_facet_attributes[
+                        'errata_counts'][errata_type],
+                    errata_type,
+                    expected_value,
+                )
+            )
+
+    def _fetch_available_errata(self, host, expected_amount, timeout=120):
+        """Fetch available errata for host."""
+        errata = host.errata()
+        for _ in range(timeout / 5):
+            if len(errata['results']) == expected_amount:
+                return errata['results']
+            sleep(5)
+            errata = host.errata()
+        else:
+            self.fail(
+                u'Host {0} contains {1} available errata, but expected to '
+                'contain {2} of them'.format(
+                    host.name,
+                    len(errata['results']),
+                    expected_amount,
+                )
+            )
+
     @tier3
-    def test_positive_install(self):
+    def test_positive_install_in_hc(self):
```

Comment:
`test_positive_install` was mentioning only about installing in host collection. decided to rename it to `test_positive_install_in_hc` and add similar test for installing in single host